src/process_wrap.cc

```diff
@@ -292,7 +292,7 @@ class ProcessWrap : public HandleWrap {
   static void OnExit(uv_process_t* handle,
                      int64_t exit_status,
                      int term_signal) {
-    ProcessWrap* wrap = static_cast<ProcessWrap*>(handle->data);
+    ProcessWrap* wrap = ContainerOf(&ProcessWrap::process_, handle);
     CHECK_NOT_NULL(wrap);
```

Comment:
Fwiw, this `CHECK` becomes somewhat pointless with this change – for `wrap` to be `nullptr`, `handle` would have to be an invalid pointer with a small offset from `nullptr` that happens to exactly match the byte offset of `process_` in `ProcessWrap` – something that’s basically guaranteed to never happen.

Reply:
@addaleax, removed `CHECK`. PTAL.